app/src/test/java/nerd/tuxmobil/fahrplan/congress/dataconverters/HighlightExtensionsTest.kt

```diff
@@ -10,11 +10,11 @@ class HighlightExtensionsTest {
     @Test
     fun toHighlightAppModel() {
         val highlightDatabaseModel = HighlightDatabaseModel(
-                sessionId = 2342,
+                sessionId = "be9979c4-18bc-52bb-9480-2c0ac2782c37",
```

Comment:
This makes it very hard for humans to compare values. If you truly need a GUID-looking value I suggest using something like `11111111-1111-1111-1111-111111111111` (`222…` etc) so humans can easily match test values.

Reply:
Good point. Thank you. :heavy_check_mark: Done in be9c1bffdb14dedf966f5f69d1c670b56fe3d04c.